knpu/mink.rst

```diff
@@ -125,7 +125,7 @@ that it extends the same base class as ``DocumentElement``, which means that
 it has almost all the same methods and more. For example, once you've found
 a ``NodeElement``, you can find more elements deeper inside of it::
 
-    $spanEle = $anchorElement.find('css', 'span.emph');
+    $spanEle = $anchorElement->find('css', 'span.emph');
```

Comment:
I would also fix the var name to `$anchorEle` to match the var name from the above... or even better fix the var name above to `$anchorElement` instead. Wdyt?